gpjax/base/module.py

```diff
@@ -192,6 +197,7 @@ def _apply_constrain(meta_leaf):
 
             return meta.get("bijector", tfb.Identity()).forward(leaf)
 
+        # TODO: Transform prior distributions to the constrained space?
```

Comment:
Could do. If it's complicated though, then just make the user define the prior on the constrained space.